packages/puppeteer-core/package.json

```diff
@@ -0,0 +1,158 @@
+    }
+  },
+  "publishConfig": {
+    "registry": "https://wombat-dressing-room.appspot.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/puppeteer/puppeteer/tree/main/packages/puppeteer-core"
+  },
+  "engines": {
+    "node": ">=14.1.0"
+  },
+  "scripts": {
+    "build:tsc": "wireit",
+    "build:types": "wireit",
+    "build": "wireit",
+    "bundle:third_party": "wireit",
+    "check": "tsx tools/ensure-correct-devtools-protocol-package",
+    "clean": "tsc -b --clean && rimraf lib src/generated",
+    "format:types": "wireit",
+    "generate:package-json": "wireit",
+    "generate:sources": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "bundle:third_party",
+        "format:types",
+        "generate:package-json"
+      ]
+    },
+    "generate:sources": {
+      "command": "tsx tools/generate_sources.ts",
+      "files": [
+        "tools/generate_sources.ts",
+        "src/templates/**"
+      ],
+      "output": [
+        "src/generated/**",
+        "src/types.ts"
+      ]
+    },
+    "bundle:third_party": {
+      "command": "rollup --config rollup.third_party.config.js",
+      "dependencies": [
+        "build:tsc"
+      ],
+      "clean": false,
+      "files": [
+        "lib/esm/third_party/**",
+        "lib/cjs/third_party/**"
+      ],
+      "output": [
+        "lib/esm/third_party/**",
+        "lib/cjs/third_party/**"
+      ]
+    },
+    "generate:package-json": {
+      "command": "echo '{\"type\": \"module\"}' > lib/esm/package.json",
+      "clean": "if-file-deleted",
+      "dependencies": [
+        "build:tsc"
+      ],
+      "output": [
+        "lib/esm/package.json"
+      ]
+    },
+    "build:types": {
+      "command": "api-extractor run --local",
+      "dependencies": [
+        "build:tsc"
+      ],
+      "files": [
+        "tsconfig.json",
+        "api-extractor.json",
+        "lib/esm/puppeteer/types.d.ts"
+      ],
+      "output": [
+        "lib/types.d.ts"
+      ]
+    },
+    "format:types": {
+      "command": "eslint --cache-location .eslintcache --cache --ext=ts --no-ignore --no-eslintrc -c=../../.eslintrc.types.cjs --fix lib/types.d.ts",
+      "dependencies": [
+        "build:types"
+      ],
+      "clean": false,
+      "files": [
+        "lib/types.d.ts",
+        "../../.eslintrc.types.cjs"
+      ],
+      "output": [
+        "lib/types.d.ts"
+      ]
+    },
+    "build:tsc": {
+      "command": "tsc -b",
+      "clean": "if-file-deleted",
+      "dependencies": [
+        "generate:sources"
+      ],
+      "files": [
+        "src/**",
+        "compat/**",
+        "**/tsconfig.*.json"
+      ],
+      "output": [
+        "lib/esm/**",
+        "lib/cjs/**"
+      ]
+    }
+  },
+  "files": [
+    "lib",
+    "!*.tsbuildinfo"
+  ],
+  "author": "The Chromium Authors",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "devtools-protocol": "0.0.1036444"
+  },
+  "peerDependencies": {
+    "ws": "8.9.0",
```

Comment:
why are these dependencies moved to the peerDependencies? I don't think we want to force the user to install those separately? Or is there some lerna feature that makes them to be prod dependencies when publishing?